plugins/warp/build.rs

```diff
@@ -18,10 +19,48 @@ fn compile_rust(file: PathBuf) -> bool {
 }
 
 fn main() {
-    if let Some(link_path) = option_env!("BINARYNINJADIR") {
+
+    let binja_base_dir: &str = option_env!("DEP_BINARYNINJACORE_BASE_DIR").unwrap_or_else(|| {
```

Comment:
Our build system needs to set its own paths, so unfortunately we can't do lookups like this.

Reply:
This would work fine: https://github.com/Vector35/binaryninja-api/commit/5239ad6e8345e691fde31fd9f513fcf200e942aa